go.mod

```diff
@@ -58,6 +58,7 @@ require (
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
 	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
 	github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96 // indirect
+	github.com/emicklei/go-restful v2.9.5+incompatible // indirect
```

Comment:
Is this a deliberate change?

Reply:
After changing the CRD, I have executed: `FORCE=true scripts/update-codegen.sh`, and I
believe it could have caused those extra mod changes.